osbrain/agent.py

```diff
@@ -898,6 +898,51 @@ def _connect_and_register(self, client_address, alias=None, handler=None,
         self.register(socket, register_as, alias, handler)
         return client_address
 
+    def unsubscribe_from_topic(self,
+                               socket: Union[AgentAddress, str, zmq.Socket],
+                               topic: bytes):
```

Comment:
We do allow `Union[bytes, str]` when making subscriptions (see Agent's `_subscribe` method). I think we should do the same here as well. User can directly use `str` (it will automatically be converted to bytes encoded in UTF8, but can use bytes if they prefer to.